infer/lib/audio.py

```diff
@@ -1,18 +1,24 @@
-import platform
+from io import BufferedWriter, BytesIO
+from pathlib import Path
+from typing import Any, Dict
 import ffmpeg
 import numpy as np
 import av
 
+video_format_dict: Dict[str, str] = {
+    "m4a": "mp4",
+}
 
-def wav2(i, o, format):
+audio_format_dict: Dict[str, str] = {
+    "ogg": "libvorbis",
+    "mp4": "aac",
+}
+
+def wav2(i: BytesIO, o: BufferedWriter, format: str):
     inp = av.open(i, "r")
-    if format == "m4a":
-        format = "mp4"
+    format = video_format_dict.get(format, "mp4")

```

Comment:
Well, well, just `get(format, format)` 😂. You can run a test on this grammar.

Reply:
oh... OHHHHHH
Sorry :sweat_smile: